docs/modules/ROOT/pages/how-to-guides/proc_maintaining_operator_graphs.adoc

```diff
@@ -0,0 +1,243 @@
+:_content-type: PROCEDURE
+:myfunctionone: maintaining_operator_graphs
+
+[id="maintaining_operator_graphs_{context}"]
+= Maintaining an Operator graph
+
+{ProductName} uses File-based catalogs instead of SQLite or ClusterServiceVersion (CVS) to produce an Operator catalog. File-based catalogs (FBC), often referred to as “Operator catalogs”, provide a simpler way to store and share metadata about Operator graphs written for Operator Lifecycle Manager (OLM). With {ProductName}, you can easily build, test, and release multiple components from a single Operator catalog repository.
+
+{ProductName} only supports use of Operator catalogs to store the Operator graphs because of the following benefits and because they have been supported since OCP 4.11:
```

Comment:
This sentence and the following bullets seem to be about highlighting the differences/benefits that FBCs specifically provide (and the specific FBC-only support by App Studio) so I wouldn't go with the generic "Operator catalog" for this part so much. Also, this is now the first-instance of "OCP", so it should now be spelled out/abbreviated here (and not in the later, previously-discussed spot).

```suggestion
{ProductName} only supports use of FBCs to store the Operator graphs because of the following benefits and because they have been supported since OpenShift Container Platform (OCP) 4.11:
```